.github/workflows/dapps.yml

```diff
@@ -105,525 +105,25 @@ jobs:
-            -e NEON_ACCOUNTS=${{ env.ACCOUNTS }} \
-            -e NEON_CHAIN_ID=${{ env.NETWORK_ID }} \
-            ${{ env.IMAGE }}
-          docker cp pancake-${{ github.run_id }}:/usr/src/app/report.json pancake-report.json
-          docker rm -f pancake-${{ github.run_id }}
-
-      - name: Save report
-        uses: actions/upload-artifact@v3
-        with:
-          name: pancake-report
-          path: pancake-report.json
-
-  swap-report:
-    runs-on: ubuntu-20.04
-    needs: [prepare, uniswap-v2, uniswap-v3, saddle, aave, curve, robonomics, yearn, compound, curve-factory, pancake]
-    if: always()
-    env:
-      SOLANA_URL: ${{ needs.prepare.outputs.solana_url }}
-      PROXY_URL: ${{ needs.prepare.outputs.proxy_url }}
-      NETWORK_ID: ${{ needs.prepare.outputs.network_id }}
-      NETWORK: ${{ needs.prepare.outputs.network }}
-      SOLANA_IP: ${{ needs.prepare.outputs.solana_ip }}
-      PROXY_IP: ${{ needs.prepare.outputs.proxy_ip }}
-    steps:
-      - uses: actions/checkout@v4
-      - name: Install python requirements
-        id: requirements
-        uses: ./.github/actions/python-requirements
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: uniswap-v2-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: uniswap-v3-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: saddle-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: yearn-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: aave-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: compound-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: curve-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: curve-factory-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: robonomics-report
-          path: reports/
-      - uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: pancake-report
-          path: reports/
-      - name: "Swap report"
-        run: |
-          python3 ./clickfile.py dapps report --directory=reports \
-          --pr_url_for_report=${{ github.event.inputs.pr_url_for_report }} \
-          --token=${{secrets.GHTOKEN}}
+    uses: ./.github/workflows/dapps_reusable.yml
+    with:
+      network: custom
```

Comment:
terraform for pipeline in neon-tests
custom for pipeline in proxy

Reply:
Changed to terraform